config/initializers/devise.rb

```diff
@@ -69,7 +69,7 @@
   # config.http_authenticatable_on_xhr = true
 
   # The realm used in Http Basic Authentication. "Application" by default.
-  config.http_authentication_realm = "Sign-On-O-Tron"
+  config.http_authentication_realm = "Signon"
```

Comment:
I had a [search through the alphagov organisation](https://github.com/search?q=org%3Aalphagov%20Sign-On-O-Tron&type=code) and I couldn't see anywhere where this realm was being used explicitly in other codebases. That doesn't mean it's not being set in, for example, an ENV variable or something but I agree it's unlikely!

Reply:
Thanks, Chris!